### PR TITLE
release 8.3.0

### PR DESCRIPTION
**Pull request by baurmatt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,26 @@ All notable changes to this project will be documented in this file.
...
 These should not affect the functionality of the module.
 
-## [v8.2.1](https://github.com/voxpupuli/puppet-prometheus/tree/v8.2.1) (2020-01-25)
+## [v8.2.2](https://github.com/voxpupuli/puppet-prometheus/tree/v8.2.2) (2020-02-28)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-prometheus/compare/v8.2.1...v8.2.2)
+
+**Implemented enhancements:**
+
+- Add service\_name parameter to haproxy-exporter [\#428](https://github.com/voxpupuli/puppet-prometheus/pull/428) ([unki](https://github.com/unki))
```
Comment (Dan33l) [inline CHANGELOG.md:13]: This PR should update Y in x.y.z versioning 
https://semver.org/ :
> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API.

Comment (baurmatt) [reply CHANGELOG.md:13]: Good catch, thank you! :)

**Comment by bastelfreak** [comment]
@baurmatt please run `bundle exec rake reference` to regenerate the REFERENCE.md and include the file in the release.

**Comment by baurmatt** [comment]
@bastelfreak Makes sense, thanks! :)

**Review by Dan33l — APPROVED** [review]
LGTM

**Pull request merged**